addons/nuxeo-platform-3d/nuxeo-platform-3d-core/src/main/java/org/nuxeo/ecm/platform/threed/service/ThreeDBatchUpdateWork.java

```diff
@@ -69,6 +69,9 @@
 
     public static final String THREED_CONVERSIONS_DONE_EVENT = "threeDConversionsDone";
 
+    // @since 11.1
+    protected static final String STATUS_DONE = "Done";
+
```

Comment:
Shouldn't this be in the previous commit, related to 3D?

Reply:
You are right, moved to the previous commit on 10.10, unfortunattely this is already merged on master